lib/lexer_mdx.mll

```diff
@@ -16,17 +16,21 @@ let labels l =
     failwith msg
 }
 
-let eol = '\n' | eof
-let ws = ' ' | '\t'
+let eol = '\n' | '\r' '\n' | eof
```

Comment:
I'm just curious, is there a specific reason not to write it like this:
```suggestion
let eol = '\n' | "\r\n" | eof
```

Reply:
No particular reason.